lms/djangoapps/courseware/access.py

```diff
@@ -146,13 +158,19 @@ def _can_access_descriptor_with_start_date(user, descriptor, course_key):  # pyl
 
     Arguments:
         user (User): the user whose descriptor access we are checking.
-        descriptor (AType): the descriptor for which we are checking access.
-    where AType is any descriptor that has the attributes .location and
-        .days_early_for_beta
+        descriptor (AType): the descriptor for which we are checking access,
+            where AType is CourseDescriptor, CourseOverview, or any other class that
+            represents a descriptor and has the attributes .location, .id, .start,
+            and .days_early_for_beta.
+
+    Returns:
+        AccessResponse: The result of this access check. Possible results are access granted or a StartDateError.
     """
+    # pylint: disable=translation-of-non-string
```

Comment:
`start_message = _(descriptor.advertised_start)`

and yeah, looks like an edX thing